app/src/androidTest/kotlin/at/bitfire/davdroid/settings/migration/AccountSettingsMigration21Test.kt

```diff
@@ -91,8 +83,8 @@ class AccountSettingsMigration21Test {
             inPendingState.first { it }
         }
 
-        // Assert again that we are now in the forever pending state
-        assertTrue(ContentResolver.isSyncPending(account, authority))
+        // Ensure that we are now in the forever pending state; Skip the test otherwise
+        Assume.assumeTrue(ContentResolver.isSyncPending(account, authority))
```

Comment:
I wonder whether we really want to use `Assume` if we still haven't tried all other methods (like making the flow hot).

The danger is that we don't know what's going on anymore, and at some point the test may even start to be always skipped (because the assumption is always false). Then it would be quite useless.

I'm for adding the `Assume` in a another PR when making the flow hot doesn't fix it.